docs/docs/guides/trainings/supportservice/operations.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Operations Trainings
+---
+
+## ORBOS Basics
+
+In this hands-on training you will gain an understanding about GitOps and ORBOS and learn how to install and manage ORBOS. We will share best practices around installation, management and configuration.
+
+**Audience**: DevOps Engineer  
+**Duration**: 1.5 days
+
+**Topics covered**:
+
+* GitOps
+* ORBITER
+  * Providers
+  * Clusters
+  * Components of an ORBOS K8s-cluster
+* BOOM
+  * Components and Integration
+* Interaction with the cluster via orbctl
+* Configuration
+* Q&A
+
+**Price**: [Get a quote](https://zitadel.ch/contact)
+
+## ZITADEL and DB Operator 
+
+In this hands-on training you will gain an in-depth understanding of the ZITADEL and DB Operators with our standard toolset.
+
+**Audience**: DevOps Engineer  
+**Duration**: 1.5 days
+
+**Topics covered**:
+
+* ZITADEL Operator
+  * Setup-process of ZITADEL
+  * Necessary configuration
+* Interaction with the cluster via zitadelctl
+* DB Operator
+  * Backups with ORBOS
+  * Secure connections
+* Common root cause analysis
+* Q&A
+
+**Price**: [Get a quote](https://zitadel.ch/contact)
+
+## ORBOS Tooling
+
+In this training you will learn how to customize tooling or more detailed aspects of the product. This is optional when using our recommended tooling and configuration to operate ZITADEL in standard scenarios.
+
+**Audience**: DevOps Engineer  
+**Duration**: 1.5 days
+
+**Topics covered**:
+
+* Metrics
+* Logs
+* Monitoring
+* Alerting
+* Ingress/API-Gateway
+* GitOps
```

Comment:
We provide a topic called gitops already with the basics, we should leave this here